ocaml-lsp-server/src/code_actions/action_destruct.ml

```diff
@@ -34,9 +36,15 @@ let code_action doc (params : CodeActionParams.t) =
       let finish = Position.logical params.range.end_ in
       Query_protocol.Case_analysis (start, finish)
     in
-    let+ res = Document.dispatch doc command in
+    let* res = Document.dispatch doc command in
     match res with
-    | Ok res -> Some (code_action_of_case_analysis doc uri res)
+    | Ok (loc, newText) -> (
+      let+ formattedText =
+        Ocamlformat_rpc.(format_type state.ocamlformat_rpc ~typ:newText)
+      in
+      match formattedText with
```

Comment:
`formatted_text`

Reply:
I'll update my variable names. Should `newText` also be renamed to `new_text`, or has this variable a reason to be formatted in camelCase?

Reply:
camelCase naming is fine if used later for named argument punning, I believe